README.md

```diff
@@ -2,6 +2,9 @@
 [![Contributing][contributing-shield]][contributing-link]
 [![Code of Conduct][conduct-shield]][conduct-link]
 [![Zulip][zulip-shield]][zulip-link]
+[![vs-marketplace-shield]][vs-marketplace-link]
+[![open-vsx-shield]][open-vsx-link]
```

Comment:
This is needed to make this proper functional markdown. If you skip the description, you still need to put the opening and closing brackets.

```suggestion
[![Available on the VsCode marketplace][vs-marketplace-shield]][vs-marketplace-link]
[![Available on the Open VSX marketplace][open-vsx-shield]][open-vsx-link]
```

Reply:
Oh, I was using the "Github renders it" excuse (similar to why almost all html in the web is broken). Thanks.